src/ui/components/remote-time-block.svelte

```diff
@@ -44,5 +55,41 @@
     left: 0;
 
     width: var(--size-4-2);
+
+    background-size: 10px 10px;
+  }
+
+  /* Striped pattern for tentative status */
+  .striped {
+    background-image: repeating-linear-gradient(
+      45deg,
+      rgb(0 0 0 / 5%),
+      rgb(0 0 0 / 5%) 5px,
+      rgb(255 255 255 / 0%) 5px,
+      rgb(255 255 255 / 0%) 10px
+    );
+  }
+
+  /* Crosshatched pattern for declined status */
+  .crosshatched {
+    background-image: repeating-linear-gradient(
+        45deg,
+        rgb(0 0 0 / 5%),
+        rgb(0 0 0 / 5%) 5px,
+        rgb(255 255 255 / 0%) 5px,
+        rgb(255 255 255 / 0%) 10px
+      ),
+      repeating-linear-gradient(
+        -45deg,
+        rgb(0 0 0 / 5%),
+        rgb(0 0 0 / 5%) 5px,
+        rgb(255 255 255 / 0%) 5px,
+        rgb(255 255 255 / 0%) 10px
+      );
+  }
+
+  .text {
+    z-index: 1;
+    padding-left: var(--size-4-2);
```

Comment:
I think it's better to style the ribbon, not the whole block. This is what people using Teams and Outlook are used to, it also looks much cleaner.
1. Use an empty ribbon with a colorful border for declined events
2. Use a ribbon with colored stripes for tentative events

Example from teams:
![image](https://github.com/user-attachments/assets/115afaf8-a6e7-4ece-a091-59caa777ba86)